terminusdb_client/tests/test_woqlQuery.py

```diff
@@ -482,10 +482,28 @@ def test_isa_method(self):
         assert woql_object.to_dict() == WOQL_JSON["isAJson"]
 
     def test_delete_method(self):
-        woql_object = WOQLQuery().delete_object("x")
+        woql_object = WOQLQuery().delete_document("x")
         json_obj = {
-            "@type": "DeleteObject",
-            "document_uri": {"@type": "NodeValue", "node": "x"},
+            "@type": "DeleteDocument",
+            "identifier": {"@type": "NodeValue", "node": "x"},
+        }
+        assert woql_object.to_dict() == json_obj
+
+    def test_update_method(self):
+        woql_object = WOQLQuery().update_document("x", "iri")
+        json_obj = {
+            "@type": "UpdateDocument",
+            "document": {"@type": "DataValue", "node": "x"},
+            "identifier": {"@type": "NodeValue", "node": "iri"},
+        }
+        assert woql_object.to_dict() == json_obj
+
+    def test_read_method(self):
+        woql_object = WOQLQuery().read_document("iri", "output")
+        json_obj = {
+            "@type": "ReadDocument",
+            "document": {"@type": "DataValue", "node": "output"},
```

Comment:
"document" actually needs to be "Value". I think it might not be! We might have to fix the definition of read_document if it isn't

Reply:
so it needs to be like this `document': {'@type': 'Value', 'node': 'output'}}` ?